recipes/gnss-sdr/meta.yaml

```diff
@@ -0,0 +1,142 @@
+{% set version = "0.0.18" %}
+
+package:
+  name: gnss-sdr-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/gnss-sdr/gnss-sdr/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d5533370de188d50c214a5bb7832a433ceb09369c3e3296a4fa34fa1aefb685e
+  patches:
+    - 0001-Manually-specify-compiled-in-GNSSSDR_INSTALL_DIR.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+outputs:
```

Comment:
SUGGESTION ONLY: It may be simpler to build once and [separate the output using glob name matching](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#specifying-files-to-include-in-output) instead of building multiple times.

Reply:
I did consider that because it's how I'd normally do outputs, but the way that their CMakeLists.txt is structured makes that more awkward. As it is, the build script for the volk-gnss-sdr output descends to the appropriate subdirectory and only builds that component, and then the subsequent build for gnss-sdr finds the already built volk-gnss-sdr library and just uses it, so there's no extra building happening.